examples/index.html

```diff
@@ -68,6 +69,7 @@ <h1>Demo pages</h1>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-number/demo">pfe-number</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-page-status/demo">pfe-page-status</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-progress-indicator/demo">pfe-progress-indicator</a></pfe-cta>
+				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-rhd-search/demo">pfe-rhd-search</a></pfe-cta>
```

Comment:
Can you remove this when you get a chance?

Reply:
I have made the suggested change